src/android/SimpleCameraPreview.java

```diff
@@ -477,13 +478,25 @@ public void run() {
         }
     }
 
-    private boolean switchCameraTo(String device, CallbackContext callbackContext) {
+    private boolean switchCameraTo(JSONObject options, CallbackContext callbackContext) {
         if (fragment == null) {
             callbackContext.error("Camera is closed, cannot switch camera");
             return true;
         }
+        int cameraDirection;
+        try {
+            cameraDirection = options.getString("direction").equals("front") ? CameraSelector.LENS_FACING_FRONT : CameraSelector.LENS_FACING_BACK;
+        } catch (JSONException e) {
+            cameraDirection = CameraSelector.LENS_FACING_BACK;
+        }
```

Comment:
same piece of code above we put it in a method, a private one maybe, along with the try catch we just return the value?